ISISScientificComputing/autoreduce
# Cycle Web Scraper Script

### Summary of work
A web scraping script to retrieve cycle data from `https://www.isis.stfc.ac.uk/Pages/Beam-Status.aspx` for use in MySQL queries to monitor Autoreductions system performance

### How to test your work
- Test script runs with no errors both with the correct url passed and then with a fake url to make sure the script is able to retrieve a local copy of cycle dates placed in a .csv file. 
- Add `print(DATA_FRAME)` to the end of the script and run from your terminal to ensure that the start and end columns have been modified accordingly. 
- Check all unit tests pass

Fixes #417
